Dockerfile

```diff
@@ -17,8 +17,3 @@ RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local
 
 RUN pecl install xdebug \
  && docker-php-ext-enable xdebug
-
-COPY composer.* ./
-RUN composer install $COMPOSER_FLAGS --no-scripts --no-autoloader
-COPY . .
-RUN composer install $COMPOSER_FLAGS
```

Comment:
Sjednoceni s tim, jak pracujem s libkama v plarform-libraries. Neni potreba to hackovat, kdyz to clvoek chce lokalne zbuildit poprvy